### PR TITLE
enhance: Remove multiple vector field limit

### PR DESCRIPTION
**Pull request by xige-16**
issue: https://github.com/milvus-io/milvus/issues/25639

/kind improvement
Signed-off-by: xige-16 <xi.ge@zilliz.com>

### CODE REVIEW

File: internal/proto/query_coord.proto [hunk 1]
```diff
@@ -213,6 +213,7 @@ message LoadMetaInfo {
   LoadType load_type = 1;
   int64 collectionID = 2;
   repeated int64 partitionIDs = 3;
+  // Deprecated
```
Comment (jiaoew1991) [inline internal/proto/query_coord.proto:216]: `string metric_type=4  [deprecated=true];` 

The code generated by this grammar is more native.

**Comment by xige-16** [comment]
rerun ut

**Comment by xige-16** [comment]
rerun ut

**Comment by xige-16** [comment]
rerun ut

**Review by longjiquan — APPROVED** [review]
/lgtm

**Review by longjiquan — APPROVED** [review]
/lgtm

**Pull request merged**